app/mstool/Parameters.cpp

```diff
@@ -63,6 +66,10 @@ void Parameters::parse_options(int argc, char* argv[])
     }
 
     logging::add_console(static_cast<severity_level>(log_level));
+    if (vm.count("log-file")) {
+        logging::add_file(log_file, static_cast<severity_level>(log_level));
+        L_(info) << "Logging output to " << log_file;
```

Comment:
Will this not just simply state that messages go to a file on the console and in the file and then continue to write the same messages to both targets? Should it not be possible to specify different severities, or maybe just different default severities?